pylib/zeus/runner.py

```diff
@@ -220,10 +227,10 @@ def _RunSingeTask(cls, task):
     TermColor.VInfo(4, 'VARS: \n%s' % task_vars)
 
     task_cmd = task
-    pipe_output = True
+    pipe_output = not Flags.ARGS.single_threaded
```

Comment:
We're assuming that we're always debugging when running single_threaded - it's probably a safe assumption so that's fine. I do think we need a comment in the code though.
Something like:
```suggestion
    pipe_output = not Flags.ARGS.single_threaded # Print to console for debugging single threaded
```